batcher/message_batcher.go

```diff
@@ -0,0 +1,145 @@
+package batcher
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+)
+
+type Sync interface {
+	SendBatch(batch [][]byte, largestSeq *big.Int, largetsSubSeq int)
+}
```

Comment:
comments describing this interface would be helpful, specifically who are the actors being synchronized, and what do the arguments mean (e.g. is it the largestSequence number contained in the batch? Seen by one of the actors?)